modules/caddytls/values.go

```diff
@@ -115,6 +115,8 @@ var defaultCurves = []tls.CurveID{
 
 // SupportedProtocols is a map of supported protocols.
 var SupportedProtocols = map[string]uint16{
+	"tls1.0": tls.VersionTLS10,
+	"tls1.1": tls.VersionTLS11,
```

Comment:
We specifically removed these along with Caddy v2 because we don't want to make it possible to serve insecure TLS versions. We want to make it as hard as possible for people to do bad things.